cmd/kubeadm/app/cmd/init.go

```diff
@@ -277,14 +277,21 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
-	// PHASE 1: Generate certificates
-	if err := certsphase.CreatePKIAssets(i.cfg); err != nil {
-		return err
-	}
+	if res, err := certsphase.UsingExternalCA(i.cfg); !res {
```

Comment:
err seems to be ignored here?

Reply:
I'd say its not an error at this step, so I changed it to be a string representing the reason that we are not in "Using External CA".